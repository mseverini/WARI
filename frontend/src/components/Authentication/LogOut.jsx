import React from 'react'
import {Redirect} from 'react-router-dom'

const LogOut = props => {
  global.sessionStorage.removeItem('token')

  return <Redirect to="/"/>
}

export default LogOut
