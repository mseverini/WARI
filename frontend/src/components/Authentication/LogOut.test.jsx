import React from 'react'
import {shallow} from 'enzyme'
import LogOut from './LogOut'

test("Renders the Login component", ()=>{
  let actual = shallow(<LogOut/>)
  expect(actual).toMatchSnapshot()
})

