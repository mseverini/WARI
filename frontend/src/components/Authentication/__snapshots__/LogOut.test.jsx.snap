// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Login component 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Redirect
    push={false}
    to="/"
/>,
  "nodes": Array [
    <Redirect
      push={false}
      to="/"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <LogOut />,
    "_instance": null,
    "_newState": null,
    "_rendered": <Redirect
      push={false}
      to="/"
/>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <LogOut />,
}
`;
